crates/pallet-feeds/src/lib.rs

```diff
@@ -121,7 +121,7 @@ mod pallet {
         /// New object was added.
         ObjectSubmitted {
             metadata: FeedMetadata,
-            who: T::AccountId,
+            feed_id: T::FeedId,
```

Comment:
Wouldn't it be helpful to have both feed ID and owner?

Reply:
This is already implied since put only allows objects from owners